src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -111,5 +139,53 @@ void getStations() {
     @DisplayName("지하철역을 제거한다.")
     @Test
     void deleteStation() {
+        // given
+        List<String> names = Arrays.asList("강남역", "역삼역");
+        saveStations(names);
+        List<Long> ids = fetchStations()
+            .jsonPath()
+            .getList("id", Long.class);
+
+        // when
+        // 강남역 제거
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+            .when().delete("/stations/" + ids.get(0))
+            .then().log().all()
+            .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+        assertThat(fetchStations()
```

Comment:
검증 대상을 변수로 분리해주시면 무엇을 검증하는지 한 눈에 더 잘 들어올 것 같습니다.
```suggestion
        List<String> stationNames = fetchStations()
            .jsonPath()
            .getList("name", String.class);
        assertThat(stationsNames).doesNotContain("강남역")
```

Reply:
`stream->filter->noneMatch` 가 직관적으로 이해하기 어려웠는데 말씀해주신 것처럼 수정하니 가독성이 많이 올라갔네요
리뷰 감사드립니다 👍 👍